03_model/src/run_model.py

```diff
@@ -674,8 +674,8 @@ def test_hyperparameters():
                                val_start_date, val_end_date, test_start_date, test_end_date, 
                                seq_len, offset, out_dir)
         
-        if os.path.exists(os.path.join(out_dir,'prepped_model_io_data')):
-           prepped_model_io_data_file = os.path.join(out_dir,'prepped_model_io_data')
+        if os.path.exists(os.path.join(out_dir, run_id, 'prepped_model_io_data')):
+           prepped_model_io_data_file = os.path.join(out_dir, run_id,'prepped_model_io_data')
```

Comment:
Since this is within the function `test_hyperparamters()` we need to add the line `run_id = hp_config['run_id']` at the beginning of the `test_hyperparamters()` function (around line 633) then uncomment out the `run_id` parameter from the `03_model/hyperparameter_config.yaml` file. I wasn't sure what the best way to do that within the PR was so I just described it, but happy to add it in myself

Reply:
Since `run_id` was actually defined as a global variable (outside any function), it is accessible inside all functions. But it is good practice to be explicit about inputs, so I will update this!

Reply:
Oh I see, no reason that I can think of, I think it makes sense to pull from the `model_config.yaml` file then

Reply:
Ok i put it in as the default in as the default input for test_hyperparameters function, and also explicitly specified that in the Snakefile

Reply:
Feel free to merge this if you think it looks ok now!